code/modules/research/designs/medical_designs.dm

```diff
@@ -115,6 +115,17 @@
 	build_path = /obj/item/device/extinguisher_refill
 	category = list("Medical Designs")
 
+/datum/design/ecig
+	name = "Electronic Cigarette"
+	desc = "nobody can read these"
```

Comment:
One day I will change the protolathe so we can, and I will moderately annoyed for having to fix your descriptions!